cmake/ucx-targets.cmake.in

```diff
@@ -7,6 +7,34 @@
 get_filename_component(prefix "${CMAKE_CURRENT_LIST_DIR}/../../.." ABSOLUTE)
 set(exec_prefix "@exec_prefix@")
 
+set(_ucxx_targets_defined "")
```

Comment:
I don't think we should make any changes here that are specific for UCXX. Can we make it generic such that it's UCX-only?

Reply:
There's nothing specific to UCXX here, it's just a variable name that I messed up. I'll rename.